src/hooks/web3.ts

```diff
@@ -10,6 +10,7 @@ import { NetworkContextName } from '../constants/misc'
 export function useActiveWeb3React(): Web3ReactContextInterface<Web3Provider> {
   const context = useWeb3ReactCore<Web3Provider>()
   const contextNetwork = useWeb3ReactCore<Web3Provider>(NetworkContextName)
+  context.account = '0x012F1B0cC84DD0cb1fC81155fD4f249b7755d76b' // just some mock address to get some claim data
```

Comment:
can you move this somewhere it isn't in the original code?

Reply:
We probably don't need it anymore since our claim data is different and we have already mock data so I will just remove it. If someone needs it quickly he can do the same while developing. `context.account = '0x012F1B0cC84DD0cb1fC81155fD4f249b7755d76b'
`